frontend/src/components/cart-page/CartSummarySection.tsx

```diff
@@ -0,0 +1,66 @@
+import { useContext } from 'react'
+import { useNavigate } from 'react-router-dom';
+
+import { ShoppingCartContext } from '../../contexts/shopping-cart-context/shoppingCartContext';
+import { Box, Button, Card, CardContent, Divider, Typography } from "@mui/material";
+
+import cartStyles from './cartStyles';
+const {
+    totalCardStyle,
+    buttonContainerStyle,
+    buttonStyle,
+} = cartStyles;
+
+const CartSummarySection = () => {
+    const navigate = useNavigate();
+    const { cartItems, userSignin } = useContext(ShoppingCartContext);
+
+    const handleProceedToCheckout = () => {
+        userSignin
+            ? navigate("/shipping")
+            : navigate("/user/signin?redirect=/shipping");
```

Comment:
let's use constants for such cases, instead of just adding the strings here (especially if we need them elsewhere) 🙏🏼

Reply:
Nice Note!

Reply:
Resolved